QATCH/QModel/models/qmodel_v7/v7_render.py

```
@@ -0,0 +1,204 @@
+    if n < 16:
+        return np.zeros(n)
+    t = pd.to_numeric(df[COL_TIME], errors="coerce").to_numpy(dtype=float)
+    dt = float(np.nanmedian(np.diff(t))) or 0.005
+    sal = np.zeros(n)
+    for col in (COL_DISS, COL_FREQ):
+        if col not in df.columns:
+            continue
+        x = pd.to_numeric(df[col], errors="coerce").to_numpy(dtype=float)
+        x = np.nan_to_num(x, nan=float(np.nanmedian(x)))
+        for scale_s in DERIV_SCALES_S:
+            w = int(round(scale_s / dt))
```

Ruff (0.15.20)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)